packages/orchestration/src/orchestration-client.test.ts

```diff
@@ -111,6 +114,66 @@ describe('orchestration service client', () => {
     expect(response.getTokenUsage().completion_tokens).toEqual(9);
   });
 
+  it('calls chatCompletion with some templating configuration (without template)', async () => {
+    const config: OrchestrationModuleConfig = {
+      llm: {
+        model_name: 'gpt-4o',
+        model_params: { max_tokens: 500 }
+      },
+      templating: {
+        defaults: {
+          topic: 'Generative AI Hub'
```

Comment:
[req] Choose a different default value, to properly test that it is overwritten.